.github/workflows/updoc.yml

```diff
@@ -0,0 +1,10 @@
+name: Updoc
+
+on:
+  workflow_dispatch: {}
+
+jobs:
+  publishing-docs:
+    uses: upbound/uptest/.github/workflows/updoc.yml@main
```

Comment:
As discussed previously, we had better fix these dependencies. But we can fix all dependencies together probably via the renovate bot, so it's okay to keep this for now in my opinion.